### PR TITLE
[Multi-sig] Cancel MultiSig Motion as Owner

### PR DESCRIPTION
**Pull request by iamsamgibbs**
[See CDapp counterpart](https://github.com/JoinColony/colonyCDapp/pull/2454)

### CODE REVIEW

**Review by bassgeta — APPROVED** [review]
Nicely done, everything works, just some changes on the CDapp counterpart necessary :ship:

File: src/graphql/mutations/multiSig.graphql [hunk 1]
```diff
@@ -7,5 +7,6 @@ mutation CreateColonyMultiSig($input: CreateColonyMultiSigInput!) {
 mutation UpdateColonyMultiSig($input: UpdateColonyMultiSigInput!) {
   updateColonyMultiSig(input: $input) {
     id
+    isRejected
```
Comment (jakubcolony) [inline src/graphql/mutations/multiSig.graphql:10]: Looks like that field isn't used anywhere

**Review by jakubcolony — CHANGES_REQUESTED** [review]
Looks great @iamsamgibbs, just a couple minor comments. Nice work!

**Review by bassgeta — APPROVED** [review]
Let's close this badboi, good work :muscle:

File: src/handlers/multiSig/motionCanceled.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  if (!colonyAddress) {
+    return;
+  }
+
+  const multiSigClient = await getMultiSigClient(colonyAddress);
+
+  if (!multiSigClient) {
+    return;
+  }
```
Comment (jakubcolony) [inline src/handlers/multiSig/motionCanceled.ts:26]: I think you might be able to get the address from the event's `contractAddress` property without having to get the client.

**Review by jakubcolony — APPROVED** [review]
Left one more suggestion but otherwise looks good! 👍

**Pull request merged**